website/docs/guides/best-practices/how-we-mesh/mesh-3-implementation.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "Implementing your mesh plan"
+description: Getting started with dbt Mesh patterns
+hoverSnippet: Learn how to get started with dbt Mesh
+---
+
+As mentioned before, the key decision to migrating to a multi-project architecture is understanding how your project is already being grouped, built, and deployed. We can use this information to inform our decision to split our project apart.
+
+- **Examine your jobs** which sets of models are most often built together?
+- **Look at your lineage graph** how are models connected?
+- **Look at your selectors** defined in `selectors.yml` - how are people already defining resource groups?
+- **Talk to teams** about what sort of separation is naturally existing right now.
+  - Are there various domains people are focused on?
+  - Are there various sizes, shapes, and sources of data that get handled separately (such as click event data)?
+  - Are there people focused on separate levels of transformation, such as landing and staging data or building marts?
+
+## Add groups and access
+
+Once you have a sense of some initial groupings, the first step is to implement **group and access permissions** within a single project.
+
+- First we'll create a [group](/docs/build/groups) to define the owner of a set of models.
+
+```yml
+# in models/__groups.yml
+
+groups: 
+  - name: marketing
+    owner:
+      - name: Ben Jaffleck 
+        email: ben.jaffleck@jaffleshop.com
+```
+
+- Then, we can add models to that group using the `group:` key in the model's yml entry.
+
+```yml
+# in models/marketing/__models.yml
+
+models: 
+  - name: fct_marketing_model
+    group: marketing
+  - name: stg_marketing_model
+    group: marketing
+```
+
+- Once models are added to the group, we will **add [access](/docs/collaborate/govern/model-access) settings to the models** based on their connections between groups, *opting for the most private access that will maintain current functionality*. This means that any model that has *only* relationships to other models in the same group should be `private` , and any model that has cross-group relationships, or is a terminal node in the group DAG should be `protected` so that other parts of the DAG can continue to reference it.
```

Comment:
```suggestion
- Once you've added models to the group, you can **add [access](/docs/collaborate/govern/model-access) settings to the models** based on their connections between groups, *opting for the most private access that will maintain current functionality*. This means that any model that has *only* relationships to other models in the same group should be `private` , and any model that has cross-group relationships, or is a terminal node in the group DAG should be `protected` so that other parts of the DAG can continue to reference it.
```